--- src/schema.rs.orig	2019-10-16 22:28:19.000000000 -0400
+++ src/schema.rs	2019-10-16 22:56:28.000000000 -0400
@@ -3,12 +3,15 @@
         id -> Int4,
         description -> Text,
     }
 }
 
 table! {
+    use diesel::sql_types::*;
+    use crate::types::*;
+
     objectives (id) {
         id -> Int4,
         description -> Text,
         ts_config_name -> Regconfig,
         ts_description -> Tsvector,
     }
@@ -28,12 +31,15 @@
         objective_id -> Int4,
         tag_id -> Int4,
     }
 }
 
 table! {
+    use diesel::sql_types::*;
+    use crate::types::*;
+
     tags (id) {
         id -> Int4,
         name -> Text,
         ts_config_name -> Regconfig,
         ts_name -> Tsvector,
     }
